pkg/cmd/provisioning/alibabacloud/create-ram-users.go

```diff
@@ -37,6 +37,8 @@ type: Opaque`
 	ramPolicyType = "Custom"
 	// autoRotateStrategy will delete the earliest inactive policy version
 	autoRotateStrategy = "DeleteOldestNonDefaultVersionWhenLimitExceeded"
+	// ccoctlResourceTagKeyPrefix is the prefix of the tag key applied to the AWS resources created/shared by ccoctl
+	ccoctlResourceTagKeyPrefix = "openshift.io/ccoctl"
```

Comment:
The variable name indicates that this is related to tagging a resource. From looking at the code, this string is simply being used to place a comment/note on the RAM User details. Also, the comment for this variable talks about AWS.

Reply:
fixed the comment, thanks for the finding! and the comment is used in delete command to find out which user should be removed if no credReqDir given

Reply:
This still calls the variable \*TagKeyPrefix\*. I think it would be more clear to remove any reference to "Tag" as AliCloud doesn't presently allow tagging RAM Users. Maybe something like `ccoctlResourcePrefix` or `ccoctlResourceCommentPrefix`?

Reply:
got it, change the variable to `ccoctlResourcePrefix`